### DISCUSSION TITLE
Javascript errors

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#314] [General]
Hi Rejetto,

I noticed some errors which seems Javascript related. Probably not a serieus problem, but I am not an excellent programmer if we talk about Javascript. PSE take a look at my screenshot. It is Google Chrome and F12 to see the console (but you know al that ofcourse).

I hope that you do not mind I keep you busy. Hihi.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/943352e3-09e7-47c2-9f2f-5f0f40e01d7c)

**Comment by rejetto** [comment]
thanks for reporting!
it's a small and harmless bug in the file-icons plugin.
You can already download the fixed update (1.03).
it happened if you have the plugin installed but never configured it.

**Reply by PH5HP** [reply]
Hi Rejetto,

Thanks for the swift reply. Problem solved.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by rejetto** [comment]
i saw your server, and i see there's an unnecessary extra height in the first page.
My advice is that you don't use the "top" section, but the "before header".
then you'll probably need an extra adjustment with  `margin-bottom: 40px;` on the .box-shadow
cheers

**Reply by PH5HP** [reply]
Hi Rejetto,

I changed it and is this what you ment?

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Reply by rejetto** [reply]
yes. I supposed you didn't mean to have the extra vertical space

**Reply by PH5HP** [reply]
Hi Rejetto,

Indeed I did not want the extra space wasted and I wondered why. I never supposed that I had to use BEFORE HEADER section instead of the TOP section.

Thanks again.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Reply by rejetto** [reply]
let's say that's the easy solution, where you fit in the layout rules of the default template.
"top" is outside.
tricky things of the css world.

**Reply by rejetto** [reply]
i will rearrange the order of the sections so to incite people at using before-header

**Comment by PH5HP** [comment]
Tnx. At the moment it is not  exactly clear what section what does. Maybe some help info would help. I wondered what the exact positions were of the sections and what differences were between some of them.

Kind regards,
Henk -PH5HP - ftp.ph5hp.nl
